scania-digital-design-system/tegel
# fix(side-menu) remove extra space to the right of a collapsed side menu item in desktop

## **Styling problem**  
Adjusts the styling to remove extra space to the right of a collapsed side menu item in desktop.

## **Issue Linking:**
- **GitHub:** https://github.com/scania-digital-design-system/tegel/issues/620  

## **How to test**  
_Provide detailed steps for testing, including any necessary setup._
1. Go to https://tds-storybook.tegel.scania.com/?path=/docs/components-side-menu--default
2. Check Collapsible and Collapsed.
3. Look at the canvas.

## **Suggested test steps**
- [x] Browser testing (Chrome, Firefox)

## **Screenshots**  
This is how the bug looks like:
<img width="220" alt="side-menu" src="https://github.com/scania-digital-design-system/tegel/assets/1917147/f9ffb23a-fd91-4f92-8888-915096bbd883">

This is how it looks like when it's resolved:
<img width="226" alt="side-menu-resolved" src="https://github.com/scania-digital-design-system/tegel/assets/1917147/aa1f9235-dd18-43f3-87e1-efd9bd571fe3">
